pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -82,6 +83,8 @@ func (a *Bootstrap) Dependencies() []asset.Asset {
 		&kubeconfig.AdminInternalClient{},
 		&kubeconfig.Kubelet{},
 		&kubeconfig.LoopbackClient{},
+		&mcign.MasterIgnitionCheck{},
+		&mcign.WorkerIgnitionCheck{},
```

Comment:
Nit, `Check` sounds passive to me, although these two assets are generating machine configs.  Maybe `Convert` or something more active?

Reply:
+1. The names were placeholders (naming is hard!!), so something like "convert" makes sense.

Reply:
Yeah @staebler suggested `MasterIgnitionCustomizations` on https://github.com/openshift/installer/pull/4413 which works for me, unless there are any objections I'll update 4413 with that name and rebase this accordingly.